src/TiledArray/conversions/fuse_arrays.h

```diff
@@ -0,0 +1,220 @@
+//
+// Created by Chong Peng on 2019-05-01.
+//
+
+#ifndef SRC_MPQC_MATH_EXTERNAL_TILEDARRAY_FUSE_ARRAYS_H_
```

Comment:
header guard is still in MPQC format, need to adjust to TiledArray format, check how other TiledArray headers define header guards